modules/personalize_blocks/personalize_blocks.admin.inc

```diff
@@ -521,7 +477,7 @@ function personalize_blocks_form_submit($form, &$form_state) {
       personalize_blocks_replace_existing($form_state['values']['original_block'], $pblock);
     }
     else {
-      $form_state['redirect'] = 'admin/structure/personalize/variations/personalize-blocks';
+      $form_state['redirect'] = 'admin/structure/personalize';
     }
```

Comment:
Would probably make more sense if we sent these redirects back to the specific campaign that owned the personalize block.  The agent machine name would be in the "agent" property of $pblock (or $option_set) in other cases and then the url is just "admin/structure/personalize/manage/{$agent}/variations"